### DISCUSSION TITLE
Error 1015

### DISCUSSION THREAD

**Discussion by mikelsanvi** [GitbookIO/community#519] [0. General bugs and questions]
When my students access my GitBook pages they get the following error.

![image](https://github.com/GitbookIO/community/assets/281207/64c65e13-dc4c-41cc-9fd4-a43082e0cc63)

This error is happening since last GitBook update.

**Comment by aikoii** [comment]
Hey @mikelsanvi, I've responded to your email, we'll dig into this with a bit more info. Thanks! 🙏

**Comment by JaniceManwiller** [comment]
This also just happened to a couple of my colleagues when they tried to go to our documentation. They're able to get to it now, so seems to have been a transient thing. One reported that they got the issue when they clicked around multiple pages within a few seconds.

**Reply by aikoii** [reply]
Hey @JaniceManwiller, looks like I overlooked this one. The issue has been fixed now, so I'm going to close it. If you're still running into any problems, let me know!
